src/engine/controls/cuecontrol.cpp

```diff
@@ -70,6 +70,21 @@ inline int hotcueNumberToHotcueIndex(int hotcueNumber) {
     }
 }
 
+void appendCueHint(HintVector* pHintList, const mixxx::audio::FramePos& frame, Hint::Type type) {
+    Hint cueHint;
+    if (frame.isValid()) {
+        cueHint.frame = static_cast<SINT>(frame.toLowerFrameBoundary().value());
+        cueHint.frameCount = Hint::kFrameCountForward;
+        cueHint.type = type;
+        pHintList->append(cueHint);
+    }
```

Comment:
```suggestion
    if (!frame.isValid) {
        return;
    }
    const Hint cueHint{
        frame: static_cast<SINT>(frame.toLowerFrameBoundary().value()),
        frameCount: Hint::kFrameCountForward,
        type
    };
    pHintList->append(cueHint);
```
Improves constness. More Rusty, less C-y.